
<launch>

  <arg name="linear_scale" default="0.5"/>
  <arg name="angular_scale" default="0.5"/>

  <node pkg="keyboard_teleop" type="keyboard_teleop_node" name="keyboard_teleop" output="screen">
    <param name="linear_scale" value="$(arg linear_scale)"/>
    <param name="angular_scale" value="$(arg angular_scale)"/>
  </node>

</launch>

